scripts/tests/chiptest/test_definition.py

```diff
@@ -372,7 +372,10 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str,
                     runner.RunSubprocess(python_cmd, name='CHIP_REPL_YAML_TESTER',
                                          dependencies=[apps_register], timeout_seconds=timeout_seconds)
             else:
-                pairing_cmd = paths.chip_tool_with_python_cmd + ['pairing', 'code', TEST_NODE_ID, setupCode]
+                pairing_cmd = paths.chip_tool_with_python_cmd + \
+                    ['pairing', 'code', TEST_NODE_ID, setupCode, '--icd-registration', 'true']
+                if self.target == TestTarget.LIT_ICD and not TestRunTime.DARWIN_FRAMEWORK_TOOL_PYTHON:
```

Comment:
This check makes no sense.  `TestRunTime.DARWIN_FRAMEWORK_TOOL_PYTHON` is not a boolean.

Reply:
updated, thanks